app/eventyay/base/migrations/0002_room_visibility_fields.py

```diff
@@ -0,0 +1,61 @@
+from django.db import migrations, models
```

Comment:
There is a pending migration AFAIR, shouldn't we do that before? Is it fine to ignore that one?